drive:
  max_driving_effort: 1.0
  min_driving_effort: -1.0
  turning_p: 10.0

gate:
  stop_thresh: 0.2
  drive_fwd_thresh: 0.34
  approach_distance: 2.0

search:
  stop_thresh: 0.2
  drive_fwd_thresh: 0.34

single_fiducial:
  stop_thresh: 0.7
  fiducial_stop_threshold: 1.75

waypoint:
  stop_thresh: 0.5
  drive_fwd_thresh: 0.34
  no_fiducial: -1

recovery:
  stop_thresh: 0.2
  drive_fwd_thresh: 0.34
  recovery_distance: 1.0 # distance the rover moves back in each leg of j-turn

failure_identification:
  dataframe_max_size: 50
  test_recovery_state: False


watchdog:
  #if we count stuck_threshold instances of being stuck, publish stuck
  dataframe_threshold: 100 #size of window we are looking at for being stuck
  stuck_threshold: 50 
  angular_threshold: 0.001 
  linear_threshold: 0.55
